api/src/main/resources/messages.properties

```diff
@@ -343,6 +343,9 @@ coreapps.deletedPatient.breadcrumbLabel=Deleted Patient
 coreapps.deletedPatient.title=This patient has been deleted
 coreapps.deletedPatient.description=Contact a system administrator if this is a mistake
 
+coreapps.patientNotFound.Description =Could not get the patient from the System
```

Comment:
Shall we change this sentence to "Patient not found in the system" (Just a thought)?, otherwise it's good. Thanks @mozzy11  for your work